protostar/commands/cairo1_commands/test_rust_command.py

```diff
@@ -0,0 +1,40 @@
+from argparse import Namespace
+from pathlib import Path
+from typing import Optional
+
+from protostar.cli import ProtostarArgument, ProtostarCommand
+from protostar.protostar_exception import ProtostarException
+
+from protostar.cairo.bindings import protostar_rust_bindings
+
+
+class TestRustCommand(ProtostarCommand):
+    @property
+    def name(self) -> str:
+        return "test-rust"
+
+    @property
+    def description(self) -> str:
+        return "Executes cairo1 tests in rust."
+
+    @property
+    def example(self) -> Optional[str]:
+        return "$ protostar test-rust"
+
+    @property
+    def arguments(self):
+        return [
+            ProtostarArgument(
+                name="path",
+                description="A path to the tests that will be run",
```

Comment:
Does it have to be a path to a file? Could I pass a path to a directory?
I think it is worth specifying it (since I can't tell from this description and I would love to know xD)

Reply:
[address feeback etc](https://github.com/software-mansion/protostar/pull/1918/commits/0b1ef91db6dd4db747aa336b04f5241da132ee67)